### ISSUE TITLE
Quests: Shutting down Manaforges - Attackers moving too fast

### ISSUE THREAD

**Issue by nemmjuuk** [dalaranwow/dalaran-wow#5226]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After initiating the shutdown with a crystal, attackers spawn and run towards the center to cancel the shutdown. They move in a very fast speed, making it very hard, if not impossible to prevent them reaching the core. 

**Expected behaviour**: Tell us what should happen instead
They should move a lot slower

**Steps to reproduce the problem**:

1. Start the quest
2. Watch what happens
3. The problem is most obvious with Manaforge Ara Quest ID: 10323 but applies to the others too. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
Compare with Manaforge Ara Quest ID: 10323 video where attackers move a lot slower:
Example: https://www.youtube.com/watch?v=z8uCd0z-bQQ
**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
